paasta_tools/kubernetes_tools.py

```diff
@@ -1578,6 +1569,7 @@ def pod_disruption_budget_for_service_instance(
     service: str, instance: str, max_unavailable: Union[str, int],
 ) -> V1beta1PodDisruptionBudget:
     return V1beta1PodDisruptionBudget(
+        # TODO: the PDB name needs to be sanitised, same as the app names.
```

Comment:
This is a potential DAR in my opinion. I would prefer we fix it as early as possible that we only have one Ocean now.

Reply:
I don't think it's a potential incident, just a potential source of confusion. What matters for PDBs are the pod selectors (using labels), which are correctly configured.

Reply:
I mean, what if one day we forgot the name didn't be sanitized and use '_' in the name. Will that cause PDB can't be setup so the cluster can't be scaled down?